### ISSUE TITLE
[BUG] Ghost cars at certain boost locations. 

### ISSUE THREAD

**Issue by Flora45** [ParadoxGamingRP/bugs-and-suggestions#1532]
### Your Discord Name

Flora40

### A clear and concise description of what the bug is

For some odd reason in a very specific area of the map it seems there are ghost cars that you can never find/aren't there. I usually spend a good 20-30 minutes looking for the car before contacting staff and 10 times out of 10 they can't find it either. Luckily though it's not an issue of a widespread area. Just 300 postal area. I will post a pic of the area below. Not sure on what could be causing it but a possibility is maybe someone starting one around the same time and their car overlaps mine or something of that nature. Idk I just work here.

### Steps to Reproduce the Bug

1. Start boost 
2. Find no boost car after searching the area. 

### Expected Behavior

1. Start boost 
2. Find boost car after searching the area. 

### Screenshots/Videos

![Capture](https://github.com/ParadoxGamingRP/bugs-and-suggestions/assets/79613077/0dd69a6f-4876-4781-b503-3007e2b69175)

### Log File

[CitizenFX_log_2023-06-20T005326.log](https://github.com/ParadoxGamingRP/bugs-and-suggestions/files/11795899/CitizenFX_log_2023-06-20T005326.log)

**Comment by xXJamieXx** [comment]
Unsure how accurate this is as people have reported issues finding vehicles in this area and staff have managed to locate the vehicles so it may just be you're having an unlucky time finding the vehicle. There's also a chance the vehicle could have been taken by someone before you find it which has been observed on multiple occasions.

**Comment by Flora45** [comment]
I would agree with this other than the fact that I've had 5+ people help me look for cars in that area as well as staff and they still can't find it. I can't confirm this for a fact but from experience it seems there's only so many spots it can spawn within a circle. If it's not in any of said areas, we go from the outside of the circle in and usually find nothing. Jayce found one of the cars I reported in the southside and he found that one. But It was in a spot that we checked already. So it must've spawned late or something.
